app.py

```diff
@@ -18,6 +18,160 @@
 """This is the main script of the template project."""
 
 from template.version import __version__
+import logging
+import sys
+from datetime import date
+from datetime import datetime
+from typing import Dict
+from typing import Any
+from typing import Optional
+from urllib.parse import urlparse
 
-if __name__ == "__main__":
-    print(f"A template project with Thoth integration, v{__version__}.")
+import requests
+import click
+import yaml
+from thoth.common import init_logging
+from thoth.storages import SolverResultsStore
+
+init_logging()
+_LOGGER = logging.getLogger("thoth.prescriptions.gh_release_notes")
+_LOGGER.setLevel(logging.INFO)
```

Comment:
Will `init_logging` handle the initialization or do I need to add/remove anything?

Reply:
It will handle logging initialization for you - the only thing needed in Thoth sources is to call the function at the beginning. You can remove `_LOGGER.setLevel` call.